LIMS_IMAGE/web/shopping/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from django.contrib.auth.models import User
+from accounts.models import *
+from orders.models import Order, Package
+from laboratory.models import Test, Location
+from datetime import datetime
+
+# Create your models here.
+class ShoppingCart(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    created_at = models.DateTimeField(default=datetime.now)
+
+class CartItem(models.Model):
+    sample_type = models.CharField(max_length=100)
+    order_number = models.ForeignKey(Order, on_delete=models.CASCADE)
+    package = models.ForeignKey(Package, on_delete=models.CASCADE)
+
+    price = models.FloatField(blank=True)
+    cart = models.ForeignKey('ShoppingCart', on_delete=models.CASCADE)
+
+    TAX = 0.12

```

Comment:
why is TAX being set here? At the moment we know that the client could be using this in any number of countries where the taxes are different, therefore having TAX as a constant doesn't make sense. I think it would be better to ignore taxes for now. When the client actually implements the taking payment and shipping fees costs etc. they can implement taxes so it works for any whatever regions and tax laws will be applicable to this.
Also, tax is usually only shown right before payment anyways and we are not there yet (nor do we need to implement that part

Reply:
> why is TAX being set here? At the moment we know that the client could be using this in any number of countries where the taxes are different, therefore having TAX as a constant doesn't make sense. I think it would be better to ignore taxes for now. When the client actually implements the taking payment and shipping fees costs etc. they can implement taxes so it works for any whatever regions and tax laws will be applicable to this. Also, tax is usually only shown right before payment anyways and we are not there yet (nor do we need to implement that part

This is a good point actually. I was assuming it is kind of a placeholder, but we might as well hold of on tax entirely for now and just implement it fully dynamically at a later stage.

Reply:
![image](https://user-images.githubusercontent.com/62345938/149553986-0977491c-39ac-42cf-b778-7583739d9563.png)

Ok, this was the requirement and I was just trying to follow that.
So I just looked up tax in Canada. 
We will fix it next week.

Reply:
Taxes are different in different Provinces, so we would have to look at the user's location to calculate tax. I think most websites do it in the checkout stage anyways so we probably don't need to worry about implementing it in the cart.